src/app/file-manager/file-download-modal/file-download-modal.component.ts

```diff
@@ -58,6 +171,7 @@ export class FileDownloadModalComponent  {
   }
 
   cancel(): void {
+    BucketService.delimiter = null;
```

Comment:
I'm not sure what this is doing. As far as I can tell, `BucketService.delimiter` is unused. If it is needed, this code should call a method on `BucketService` to perform this operation, e.g. `this.bucketService.reset()`.

Reply:
looks like more remains of unused code. Will delete.